R/dust_class.R

```diff
@@ -120,5 +120,13 @@ dust_class <- R6::R6Class(
     ##' Only returns non-NULL if the model provides a `dust_info` template
     ##' specialisation.
     info = function() {
+    },
+
+    ##' @description
+    ##' Returns the state of the random number generator. This returns a
+    ##' raw vector of length 32 * n_particles. It is primarily intended for
```

Comment:
Specify that the vector is of positive/unsigned integers (and the other places this appears in docstrings)

Reply:
From the point of view in R, it's only a raw vector (they're opaque) so it *represents* unsigned integers but it's really just data at this point